src/index.ts

```diff
@@ -0,0 +1,23 @@
+import { Router, AppRouter, RouteLoader } from 'aurelia-router';
+import { TemplatingRouteLoader } from './route-loader';
+import { RouterView } from './router-view';
+import { RouteHref } from './route-href';
+import { IFrameworkConfiguration } from './interfaces';
+
+export function configure(config: any) {
```

Comment:
@bigopon why wouldn't this be config: IFrameworkConfiguration.

Reply:
yeah, it will be changed. I was probably holding it while waiting for the main PR in router